views/elements/current_and_next.tmpl

```diff
@@ -0,0 +1,20 @@
+{{define "current_and_next"}}
+
+	{{if .Current}}
```

Comment:
My bad, yes all that whitespace can be removed. I didn't know about the trim functionality :stuck_out_tongue: